api/v1alpha1/virtualmachine_conversion.go

```diff
@@ -799,6 +802,20 @@ func Convert_v1alpha3_VirtualMachineStatus_To_v1alpha1_VirtualMachineStatus(
 	out.LastRestartTime = in.LastRestartTime
 	out.Conditions = translate_v1alpha3_Conditions_To_v1alpha1_Conditions(out.Conditions)
 
+	out.Volumes = nil
+	for i := range in.Volumes {
+		if in.Volumes[i].Type != vmopv1.VirtualMachineStorageDiskTypeClassic {
+
+			// Only down-convert volume statuses if the volume is managed.
```

Comment:
Total nit - maybe change the check above to be affirmative `Type == Managed` to match this comment closer, and be clearer that this is PVCs only in case a third type comes along

Reply:
I specifically did this to handle the case where it's not yet set to `Managed` for some reason.